src/lib/bootloader/sysconfig.rb

```diff
@@ -35,14 +35,14 @@ def initialize(bootloader: nil, secure_boot: false, trusted_boot: false, update_
     end
 
     def self.from_system
-      bootloader = Yast::SCR.Read(AGENT_PATH + "LOADER_TYPE")
+      bootloader = Yast::SCR.Read("#{AGENT_PATH}LOADER_TYPE")
```

Comment:
and here is wrong auto correct. Because AGENT_PATH is not string, but Yast::Path, then result is different.